README.brazilian-portuguese.md

```diff
@@ -18,21 +18,21 @@
 
 <br/>
 
-Leia em diferentes linguagens: [![CN](/assets/flags/CN.png)**CN**](/README.chinese.md), [![BR](/assets/flags/BR.png)**BR**](/README.brazilian-portuguese.md) [(![ES](/assets/flags/ES.png)**ES**, ![FR](/assets/flags/FR.png)**FR**, ![HE](/assets/flags/HE.png)**HE**, ![KR](/assets/flags/KR.png)**KR**, ![RU](/assets/flags/RU.png)**RU** and ![TR](/assets/flags/TR.png)**TR** in progress!)](#translations)
+Leia em diferentes idiomas: [![CN](/assets/flags/CN.png)**CN**](/README.chinese.md), [![BR](/assets/flags/BR.png)**BR**](/README.brazilian-portuguese.md) [(![ES](/assets/flags/ES.png)**ES**, ![FR](/assets/flags/FR.png)**FR**, ![HE](/assets/flags/HE.png)**HE**, ![KR](/assets/flags/KR.png)**KR**, ![RU](/assets/flags/RU.png)**RU** and ![TR](/assets/flags/TR.png)**TR** in progress!)](#translations)
 
 <br/>
 
-###### Built and maintained by our [Steering Committee](#steering-committee) and [Collaborators](#collaborators)
+###### Construído e mantido pelo nosso [Comitê Diretivo](#steering-committee) e [Colaboradores](#collaborators)
 
-# Latest Best Practices and News
+# Novas Práticas e Notícias
 
-- **New translation:** ![BR](/assets/flags/BR.png) [Brazilian Portuguese](/README.brazilian-portuguese.md) available now, courtesy of [Marcelo Melo](https://github.com/marcelosdm)! ❤️
+- **Nova tradução:** ![BR](/assets/flags/BR.png) [Português Brasileiro](/README.brazilian-portuguese.md) disponível agora, cortesia de [Marcelo Melo](https://github.com/marcelosdm)! ❤️
 
-- **New best practice:** 4.2: Include 3 parts in each test name - [_From the section "Testing and overall quality"_](https://github.com/i0natan/nodebestpractices#4-testing-and-overall-quality-practices)
+- **Nova prática recomendada:** 4.2: Inclua 3 partes em cada nome de teste - [_Da seção "Teste e qualidade geral"_](https://github.com/i0natan/nodebestpractices#4-testing-and-overall-quality-practices)
 
-- **New best practice:** 7.1: Prefer native JS methods over user-land utils like Lodash - [_From the section "Performance"_](https://github.com/i0natan/nodebestpractices#7-performance-best-practices)
+- **Nova prática recomendada:** 7.1: Prefira métodos JS nativos ao invés de utilitários de usuário, como o Lodash - [Da seção "Performance"\_](https://github.com/i0natan/nodebestpractices#7-performance-best-practices)
 
-- **News update:** [We kicked-off the performance section, wanna join?](https://github.com/i0natan/nodebestpractices/issues/302)
+- **Novidades:** [Demos o pontapé inicial na seção de performance, quer fazer parte?](https://github.com/i0natan/nodebestpractices/issues/302)
 
```

Comment:
Looks like you had format-on-save turned on in your editor while editing this. 

We probably want to undo the changes past this point.

Everything else above looks great! Thanks! 👍

Reply:
@sagirk Sorry!!! You're right: I really had format-on-save turned on.

But, now I have a little doubt (a newbie one): how can I resolve this? I already changed in my local file. Shoul I just push to remote again?

Reply:
Either this or you could copy the file's current state (so only the lines below the changes you wanted to make) and simply replace your formatted content to undo those changes easily.

Reply:
Thanks @BrunoScheufler 

I've made those changes and I think we're good to go!

Sorry again for this 😣